client/client.go

```diff
@@ -35,10 +35,14 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
-func unixDialer(_, _ string) (net.Conn, error) {
+func privateUnixDialer(_, _ string) (net.Conn, error) {
```

Comment:
I think there are two perspectives here: the point of view from a snap, and the point of view from the overall system. I was looking at it from the former, where from within the snap one was available to me, the other wasn't (public and private). Calling these sockets `snapd.socket` and `snap.socket`, then, seems confusing because we have a binary called `snapd` and another binary called `snap`, which seems to imply that `snapd` uses the `snapd.socket`, and `snap` uses the `snap.socket`, which obviously isn't the case.

However, using the point of view of the system, the `snap.socket` makes sense in a "this is the socket typically used by snaps." But looking at that alongside the `snapd.socket`, the naming doesn't seem consistent with that meaning. i.e. `snapd.socket` is not "the socket that is typically used by snapd." In fact, it's the socket that is typically used by the `snap` command, which muddies the waters even more :stuck_out_tongue: .

`snapd-snap.socket` might be an improvement, though I still can't look at it and derive meaning from it. Do you like that better?